sp800-63b/sec5_authenticators.md

```diff
@@ -4,26 +4,26 @@
 
 _This section is normative._
 
-This section provides the detailed requirements specific for each type of authenticator. With the exception of reauthentication requirements specified in [Section 4](#AAL_SEC4) and the requirement for verifier impersonation resistance at AAL 3 described in section [5.2.5](#verifimpers), the technical requirements for each of the authenticator types are the same regardless of the AAL at which the authenticator is used.
+This section provides the detailed requirements specific for each type of authenticator. With the exception of reauthentication requirements specified in [Section 4](#AAL_SEC4) and the requirement for verifier impersonation resistance at AAL3 described in [Section 5.2.5](#verifimpers), the technical requirements for each of the authenticator types are the same regardless of the AAL at which the authenticator is used.
 
-### 5.1. Requirements by Authenticator Type
+### <a name="reqauthtype"></a> 5.1. Requirements by Authenticator Type
 
 #### 5.1.1. Memorized Secrets
 
 <div class="text-left" markdown="1">
 <table style="width:100%">
   <tr>
     <td><img src="sp800-63b/media/Memorized-secret.png" alt="authenticator" style="width: 100px;height: 100px;min-width:100px;min-height:100px;"/></td>
-    <td>A Memorized Secret authenticator (commonly referred to as a <i>password</i> or, if numeric, a <i>PIN</i>) is a secret value that is intended to be chosen and memorizable by the user. Memorized secrets need to be of sufficient complexity and secrecy that it would be impractical for an attacker to guess or otherwise discover the correct secret value.</td>
+    <td>A Memorized Secret authenticator (commonly referred to as a <i>password</i> or, if numeric, a <i>PIN</i>) is a secret value that is intended to be chosen and memorable by the user. Memorized secrets need to be of sufficient complexity and secrecy that it would be impractical for an attacker to guess or otherwise discover the correct secret value.</td>
   </tr>
   </table>
   </div>
 
-##### 5.1.1.1. Memorized Secret Authenticators
+#### 5.1.1.1. Memorized Secret Authenticators
```

Comment:
That's correct. However, I intentionally left it as #### because beyond that the headers get very small and hard to discern from the other text. Capping it at 4 #'s is a similar approach to what we did for NISTIR 8112.